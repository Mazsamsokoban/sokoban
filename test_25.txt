loadtest 25
move w1 right
showmaze
showstate w2