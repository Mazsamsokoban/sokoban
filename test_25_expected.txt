f1:w1 f2:box f3:w2 obs:null 

f1:null f2:w1 f3:box obs:null 

name:w2
field:null
direction:Right
points:0
pushforce:10.0

